cmd/fyne/internal/commands/package-web.go

```diff
@@ -9,60 +10,101 @@ import (
 	"fyne.io/fyne/v2/cmd/fyne/internal/util"
 )
 
-func (p *Packager) packageWeb() error {
-	appDir := util.EnsureSubDir(p.dir, "web")
+type webData struct {
+	WasmFile     string
+	GopherJSFile string
+	IsReleased   bool
+	HasWasm      bool
+	HasGopherJS  bool
+}
 
+func packageWebInternal(w webData, appDir string, exeWasmSrc string, exeJSSrc string, icon string, release bool) error {
```

Comment:
The methods are now out-of-order, functions should appear after types and their methods.
We haven't found a way to automate this check sorry!